pkg/exposer/csi_snapshot.go

```diff
@@ -488,6 +488,10 @@ func (e *csiSnapshotExposer) createBackupPod(
 	}
 	label[podGroupLabel] = podGroupSnapshot
 
+	for k, v := range thirdPartyLabels {
```

Comment:
Is there any logic to guarantee only the labels both in Velero pod and while list are added to the backup/restore pods?

Reply:
There is not this guarantee. Since it is working as a white list, no matter whether they are in Velero pods, it is always safe to use them in data mover pods. So do you think it is necessary to make them match?

Reply:
Done, added the logic to ensure the same label exists with node-agent and get the value from node-agent as well.